projects/fsastorefrontlib/src/cms-components/my-account/claim/claim-details/claim-details.component.html

```diff
@@ -95,6 +95,7 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
       <div class="accordion-list">
         <cx-fs-documents-table
           [documentSource]="claim.documents"
+          [visibleColumn]="false"
```

Comment:
Maybe to rename this property to something more meaningful like: visibleBelongingToColumn or referenceObjectColumnVisible.

Reply:
Currently it is Belonging to column, but maybe in the future it will be some other with different name so I chose more generic name. If you are sure it will not be the case I can change the name?

Reply:
As we agreed I'll change the variable name to `referredObjectColumnVisible`.